js/reducers/windows.ts

```diff
@@ -127,6 +129,26 @@ const windows = (
           }
         }
       };
+    case ENABLE_MILKDROP:
+      return {
+        ...state,
+        genWindows: {
+          ...state.genWindows,
+          [WINDOWS.MILKDROP]: {
+            title: "Milkdrop",
+            size: [7, 12],
+            open: action.open,
+            options: action.options,
+            hidden: false,
+            shade: false,
+            canResize: true,
+            canShade: false,
+            canDouble: false,
+            generic: true,
+            position: { x: 275, y: 0 }
```

Comment:
The other windows use `{x:0,y:0}`. Let's stick with that. I know it's kinda BS, since the initial position is always ignored. Sorry about that.

Reply:
no worries, sounds good!